submissions/Roka20012/pop-up-menu/style.css

```diff
@@ -55,91 +50,100 @@
   background-color: white;
 }
 
-.check {
+.show-more {
   display: none;
-  position: relative;
 }
 
-.check:checked ~ .container-menu .sprite-container,
-.check:checked ~ .mark, .check:checked ~ .container-menu .hide-label,
-.check:checked ~ .container-menu {
-  display: grid;
+.check {
+  position: absolute;
+  top: 2px;
+  left: 74%;
+  opacity: 0.6;
 }
 
-.checkmark {
+.check:hover {
+  opacity: 1;
+}
+
+.check::after {
   position: absolute;
-  opacity: 0.6;
-  top: 5px;
   width: 16px;
   height: 16px;
-  left: 75%;
   cursor: pointer;
   background-color: #888;
+  content: "";
 }
 
-.checkmark:hover {
-  opacity: 1;
-}
-
-a {
-  text-decoration: none;
-  text-align: center;
-  color: black;
-  font-family: Arial;
-  font-size: 14px;
-
+.check:focus::after {
+  box-shadow: 0px 0px 1px 1px #A8C9FEFF;
 }
 
-a:focus {
-  box-shadow: 0px 0px 1px 1px #A8C9FEFF;
+#check-show-more {
+  display: none;
 }
 
-span {
-  left: -1px;
-  position: relative;
-  top: -5px;
+.check:checked ~ #check-show-more {
+  display: grid;
 }
 
-a:hover {
-  box-shadow: 0px 0px 0px 1px #E5E5E5FF;
+#check-show-more {
+  position: absolute;
+  z-index: 1;
+  top: 367px;
+  left: -3px;
 }
 
-.show-text, .hide-text {
-  cursor: pointer;
-  color: #4F4F4FFF;
-  font-family: Arial;
-  user-select: none;
+#check-show-more::after {
   position: absolute;
-  display: grid;
-  width: 263px;
-  height: 30px;
-  box-shadow: 0px 0px 0px 1px #E5E5E5FF;
+  width: 260px;
+  height: 31px;
   padding-top: 10px;
+  cursor: pointer;
+  content: "Show More";
   text-align: center;
-  top: 329px;
+  font-family: Arial;
+  font-size: 16px;
   background-color: #E5E5E5FF;
+  color: #4F4F4FFF;
 }
 
-.hide-text {
-  display: none;
+#check-show-more:hover::after {
+  background-color: #CCC;
 }
 
-#check-show-more:checked ~ .container-menu label > .show-text {
-  display: none;
+#check-show-more:focus::after {
+  box-shadow: 0px 0px 1px 2px #A8C9FEFF;
 }
 
-.hide-text:hover {
-  text-decoration: underline;
+.check:checked ~ .container-menu .sprite-container,
+.check:checked ~ .mark,
+.check:checked ~ .container-menu .hide-label,
+.check:checked ~ .container-menu {
+  display: grid;
 }
 
-#check-show-more:checked ~ .container-menu label > .hide-text {
-  top: 560px;
+a {
```

Comment:
Avoid styling by tag in the global context.

Reply:
Already done thanks:)